packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -146,38 +158,45 @@ const setupDriver = async () => {
 		}
 	}
 
-	if ( ! isLocalEnvironment() ) {
-		desiredCaps.name = `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`;
-		desiredCaps.tags = [ 'Gutenberg', branch ];
-	}
-
-	await driver.init( desiredCaps );
+	const sauceOptionsConfig = ! isLocalEnvironment()
+		? {
+				'sauce:options': {
+					...sauceOptions,
+					name: `Gutenberg Editor Tests[${ rnPlatform }]-${ branch }`,
+					tags: [ 'Gutenberg', branch ],
+					idleTimeout: 240,
+				},
+		  }
+		: {};
+	const serverConfig = isLocalEnvironment()
+		? serverConfigs.local
+		: serverConfigs.sauce;
 
-	const status = await driver.status();
-	// Display the driver status
-	// eslint-disable-next-line no-console
-	console.log( status );
+	const driver = await remote( {
+		...serverConfig,
+		logLevel: 'error',
+		capabilities: {
+			platformName: PLATFORM_NAME,
+			...prefixKeysWithAppium( desiredCaps ),
+			...sauceOptionsConfig,
+		},
+	} );
 
 	await driver.setOrientation( 'PORTRAIT' );
 	return driver;
 };
 
 const stopDriver = async ( driver ) => {
 	if ( ! isLocalEnvironment() ) {
-		const jobID = driver.sessionID;
-
-		const hash = crypto
```

Comment:
This was causing issues due to the server configuration changes for SauceLabs. The jobs links are public so I don't think we need that auth param.